bpfd-common/src/lib.rs

```diff
@@ -23,14 +23,14 @@ unsafe impl aya::Pod for XdpDispatcherConfig {}
 // TC Defines
 pub const TC_METADATA_SECTION: &str = "tc_metadata";
 pub const TC_DISPATCHER_VERSION: u32 = 1;
-pub const TC_DISPATCHER_RETVAL: u32 = 0; //TC_ACT_OK
+pub const TC_DISPATCHER_RETVAL: u32 = 31;
```

Comment:
Is this a typo? Why 31? I don't see that as a valid return code https://elixir.bootlin.com/linux/v4.5/source/include/uapi/linux/pkt_cls.h#L82

Reply:
Not a typo - it's a special code used by the XDP dispatcher that gets returned if we've entered a stub function an extension program isn't loaded.

Reply:
What about adding a link here to the special code, so that future readers can sort of find there way to where it originates without having to have tacit knowledge on the subject?